packages/runtime/container-runtime/src/index.ts

```diff
@@ -17,17 +17,63 @@ export {
     agentSchedulerId,
     ContainerRuntime,
 } from "./containerRuntime";
-export * from "./deltaScheduler";
-export * from "./dataStoreRegistry";
+export { DeltaScheduler } from "./deltaScheduler";
+export { FluidDataStoreRegistry } from "./dataStoreRegistry";
 export {
     gcBlobPrefix,
     gcTreeKey,
     IGarbageCollectionRuntime,
     IGCStats,
     IUsedStateStats,
 } from "./garbageCollection";
-export * from "./pendingStateManager";
-export * from "./summarizer";
-export * from "./summarizerTypes";
-export * from "./summaryCollection";
+export {
+    IPendingFlush,
+    IPendingFlushMode,
+    IPendingLocalState,
+    IPendingMessage,
+    IPendingState,
+    PendingStateManager,
+} from "./pendingStateManager";
+export { Summarizer } from "./summarizer";
```

Comment:
I'm not sure pending state manager interfaces should be exported. Can they be removed?

Reply:
The public ContainerRuntime.getLocalPendingState API returns IPendingLocalState, and that winds up pulling in all of these.

Reply:
...but PendingStateManager itself doesn't need to be exported, apparently.